From 0d4440b8085149b34319783039d95673334cd63a Mon Sep 17 00:00:00 2001
From: Christopher Chavez <chrischavez@gmx.us>
Date: Fri, 18 Jan 2019 18:17:58 -0600
Subject: [PATCH] tkx-ed: remove tclCarbonProcesses usage

Recent Tk releases use Cocoa rather than Carbon,
so CarbonCritLib no longer works.
---
 tkx-ed | 10 ----------
 1 file changed, 10 deletions(-)

diff --git tkx-ed tkx-ed
index fc3cc74..3ee7097 100644
--- tkx-ed
+++ tkx-ed
@@ -20,16 +20,6 @@ if ($IS_AQUA) {
     # The console can pop up unexpectedly from the tkkit
     Tkx::catch("console hide");
 
-    # Set the process name that is displayed in the Activity Monitor
-    # and the Force Quit dialog
-    eval {
-        Tkx::package_require('tclCarbonProcesses');
-        my $psn = Tkx::carbon__getCurrentProcess();
-        Tkx::carbon__setFrontProcess($psn);
-        Tkx::carbon__setProcessName($psn, $PROGNAME);
-    };
-    warn $@ if $@;
-
     Tkx::interp_alias("", "::tk::mac::OpenDocument", "", [\&load]);
 }
 
-- 
2.17.2 (Apple Git-113)

